.github/ISSUE_TEMPLATE/bug_issue.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Issue Template
+about: Bug reporting template
+title: '[BUG]'
+labels: 'bug'
+assignees: ''
+
+---
```

Comment:
The headers are so that GitHub recognizes the file as a an issue template and places the appropriate label and assignee, etc. So it's not rendered in the issue body, but the appropriate fields contribute to other issue functions, including the title.

Reply:
See them in action here: https://github.com/jbteves/tedana/issues/new/choose

Reply:
That's odd. Why did #190 have the heading rendered as text initially then?

Reply:
Well it renders in the .md, but it doesn't render when you open an issue. Sorry that wasn't clear.

Reply:
Okay gotcha. It was because you copied the contents of the template over manually instead of it being incorporated in the repo. Makes sense, thank you.

Reply:
I'd still prefer not to alter the titles with the [label] prefix. IMO those are useful for PRs because we don't expect PRs to build up, but for issues it's easier to sort them using the labels directly. I'd also not like to hint that issue raisers should be assigning issues to anyone. If this was an internal repository, or if people were paid to work on it, then that would make more sense to me, but since it's a volunteer-driven package I think people should only self-nominate to handle specific issues.

I do love the idea of giving issue raisers the power to assign labels though. That's a great feature.

Reply:
Makes sense to get rid of the prefix since we already have the labels, yeah. That's just what my lab uses so I ported it over. The assignment isn't something we can turn off as far as I'm aware.

Reply:
Oh okay, that makes sense. 👍